frontend/src/index.tsx

```diff
@@ -48,6 +49,22 @@ const dialectReportViewDataLoader = async ({
   return { reportData, allImplementationsData };
 };
 
+const benchmarkReportViewDataLoader = async ({
+  params,
+}: {
+  params: Params<string>;
+}) => {
+  const draftName = params?.draftName ?? "draft2020-12";
```

Comment:
Rather than hardcoding `"draft2020-12"` see if we already have a `LATEST_DIALECT` constant somewhere, or if not possibly we should.

Reply:
Not really my place to comment here but I did do a minor hackaround to fix this over [here](https://github.com/bowtie-json-schema/bowtie/blob/4ae905b04eb452eb3542e31ef2b1f6e6b9431392/frontend/src/index.tsx#L47) in the trend chart PR.

Reply:
Yeah I think we can have a Dialect.latest()